services.tf

```diff
@@ -1,16 +1,7 @@
 
 resource "cloudfoundry_service_instance" "s3" {
   name         = "s3"
-  space        = cloudfoundry_space.space.id
+  space        = local.space_id
+  # TODO: This needs to be driven by the CF region
   service_plan = data.cloudfoundry_service.s3.service_plans["US Standard"]
```

Comment:
This probably needs to be configurable with a default.

Reply:
Yep, this popped up during refactoring. Ideally it should be driven by the current region but the region parameters that the brokers expects are not 1-on-1 with the mapping we use in CF, shrugh..